[READY] Adds Departmental Emergency Response Teams

SgtHunk opened the pull request:
## About The Pull Request

This Pull Request adds Departmental ERTs, who serve their own functions instead of being divided like ERTs usually are.
Due to their more focused nature, they get better tools than what a divided ERT member would get.
That is, things like:

- Engineering and Medical get Experimental Tools.
- Security gets a full webbing with tools. (Along with the Tactical Energy Gun which normal Sec ERT gets)
- Engineering and Medical Commanders get a special hardsuit with built-in SecHUDs.
- Engineering Commander gets a Combat RCD.
- Medical Commander gets a Combat Experimental Quantum Medicine Injector.
- Engineering ERT is more prepared (by wearing oxygen tank internals).
- Engineering ERT might have a 5% chance to have the urge to build a shed, provide advice to the crew about home construction along with the tools and materials to make a state of the art Home.
- Security Commander is robust (as expected of a Head of Security that arrived from CentCom), his armor blocks knockdowns from shoves, he's been outfitted with a thermal eyepatch, and his helmet also has the regular built-in SecHUDs display.
- Medical ERT (And its Commander) have been outfitted with special, insulated, Nitrile gloves for quick body carrying.
- Medical ERT Units have a 15% chance to spawn... differently. Either with ancient Rods with enchanted healing power or through being a Survivor of a Crashed Mothership.
- Mining ERT, with upgrades, advanced plasma cutters, special hardsuits, and Combat Jump Boots.
- Mining ERT Commander gets a Kinetic Crusher + trophies along with it. He has the HECK Suit palette.
- A Security ERT member has a small chance to be a synth. (And an even smaller chance to become a MILITARY one.)
- **HONK!** _They will arrive._

## Why It's Good For The Game

![image](https://user-images.githubusercontent.com/68669754/92985095-e0595500-f485-11ea-99d8-aac049db195c.png)
![image](https://user-images.githubusercontent.com/68669754/92985129-30381c00-f486-11ea-865c-c7b77235944d.png)
![image](https://user-images.githubusercontent.com/68669754/92985130-362dfd00-f486-11ea-9df7-99d7c25bfbfb.png)
![image](https://user-images.githubusercontent.com/68669754/93010220-60ec8400-f560-11ea-9428-cdbdb5706ce4.png)
![image](https://user-images.githubusercontent.com/68669754/93010223-64800b00-f560-11ea-8454-9b0299f2be0b.png)

More ERT Variety is good. Ever wanted to call an ERT specifically to combat a virus/tons of dead crew/Mutinies without annihilating with a Deathsquad/Supermatter blowing up/All Miners are fucking dead/The Clown is fucking dead/Want to maxcap medbay and blame it on ERT? Well, now's the time, these guys are going to do it for you.

## Changelog
:cl:
add: Added Departmental ERTs for the Medical, Security, and Engineering departments.
/:cl:

SgtHunk committed:
NO TG Files edited baybee

The whole thing
- Adds Departmental ERT Datums
- Adds Departmental ERT Antag Datums
- Adds Departmental ERT Outfits
- Adds Departmental ERT Commander Hardsuits
- Adds Prefilled Webbings/Medbelts for the Departmental ERTs
- hehe easter egg

GuillaumePrata commented:
> NO TG Files edited baybee

My favorite words

SgtHunk commented:
An accomplishment

SgtHunk committed:
ERT Commander Clothing and stuff

- Adds the ERT Commander hardsuit sprites
- Makes the ERT Security Commander ROBUST. (By giving him a thermal eyepatch, his hardsuit prevent knockdown from shoves, and his helmet having integrated sec-huds like the other Commander helmets.)

SgtHunk committed:
Insulated Nitrile + fixes oversight

Adds Insulated Nitrile gloves to Med ERT, also gives Sec ERT Commander guerilla gloves (instead of combat)

ErwinSnowden commented:
And we need a person who's effectively a validhunting role w/ an electrode taser and nostun? Especially with the ability for mods to authorize emergency response teams?

This doesn't even have a mining response team to deal with megafauna. Downvoted.

SgtHunk commented:
Give me sprites and I add a Mining ERT

Also normal sec ERT already gets stun weapons lmao, this just makes it so whoever calls for ERT doesn't have to call a Deathsquad instead to solve security 'issues'

SgtHunk committed:
it was a neccesary sacrifice

edits tg files
- Adds Mining ERT/Commander + Clown ERT Commander Hardsuit
- blah blah blah datums, antag datums, outfits
- mining ert gets its own icon

SgtHunk commented:
![Screen_Shot_2019-04-22_at_12.03.08_PM.jpg](https://user-images.githubusercontent.com/68669754/93010246-d0627380-f560-11ea-912d-20d904b064c6.jpg)

SgtHunk committed:
Alien Med ERT is Real

15% chance for you to be an Alien/Oath Med ERT on spawn
Oath simply gets Rod of Asclepius, Alien gets an Alien Pistol instead of Egun, a medbelt with alien tools + has the Abductor race
Also Engi ERT gets a RPD instead of mini extinguisher

SgtHunk committed:
Sec ERT is now based

1/10 chance for Sec ERT to be a synth(with yet another 1/10 to be a Mil Synth)

ATH1909 commented on code/FulpstationCode/fulp_ert.dm:
```diff
@@ -0,0 +1,670 @@
+
+/obj/item/clothing/gloves/color/latex/nitrile/infiltrator/medical
+	name = "insulated nitrile gloves"
+	desc = "Very pricy Specialized gloves, thicker than latex. Transfers intimate paramedic knowledge into the user via nanochips, along with providing insulation."
```
```suggestion
	desc = "Very pricey specialized gloves, thicker than latex. Transfers intimate paramedic knowledge into the user via nanochips, along with providing insulation."
```

trerri replied:
pricy is correct dumbass
